app/models/lookup_keys/lookup_key.rb

```diff
@@ -103,7 +103,8 @@ def supports_uniq?
   end
 
   def to_param
-    Parameterizable.parameterize("#{id}-#{key}")
+    search_key = key.tr(' ','_') unless key.nil?
```

Comment:
I was worried about that too but there is no other way to create a down migration.